en/airframes/airframe_reference.md

```diff
@@ -39,6 +39,34 @@ This page lists all supported airframes and types including
 </tr>
 </tbody></table>
 
+### Dodecarotor cox
```

Comment:
You're right, that does not exist yet. I'll see if I can get it done, I'll need to check the motor assignments & directions.